Nova-LMS-Project/src/com/lms/dao/AuthorDao.java

```diff
@@ -0,0 +1,21 @@
+package com.lms.dao;
+
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.util.List;
+
+import com.lms.model.Author;
+
+public interface AuthorDao {
+	// creates new entry using DataObject into file
```

Comment:
I recommend using Javadoc notation for explanatory comments like this, even if you don't go so far as to give something the full Javadoc treatment. (See [the Wikipedia article](https://en.wikipedia.org/wiki/Javadoc), or any other reference, for an explanation of the comment format that Javadoc looks for.)